js/core/index.js

```diff
@@ -167,6 +167,9 @@ require([
       .success(function(data) {
         config.cluster.infos = data;
         $(document).trigger('show', { page: config.STARTPAGE });
+      })
```

Comment:
I'm not sure what this change is supposed to fix, maybe the commit message would deserve few more sentences to make it clearer.

Also, I'm not sure the users should be redirected on the login page for every errors. There are several reasons for the /cluster to return error, including 500. The login form is relevant only in case of 403 and auth enabled on the cluster. In all other cases, it is pointless.